install

```diff
@@ -177,12 +177,10 @@ while [[ $# -gt 0 ]]; do
 done
 
 # Variables
-PYTHON="python3"
-PIP="$PYTHON -m pip"
-
 DEFAULT_WEB_USER="www-data"
 
 # Paths
+INSTALL_DIR="/opt/libretime"
```

Comment:
`/opt/libretime` seem like a good path for this, but maybe it doesn't comply enough with FHS ? What would be the other alternatives ?

Reply:
`/op/` seems to be a fairly standard place for software. I agree it isn't strictly FHS, but it's common enough to be a standard